peachjam/views/generic_views.py

```diff
@@ -109,6 +111,9 @@ class Group:
             def __init__(self, title):
                 self.title = title
 
+                if group_by == "date":
+                    self.group_id = months_reverse[title.split()[0]]
```

Comment:
@Sandravaphilips I've altered the logic a bit here. the previous was giving an incorrect index, eg. it gave November the id of 10, instead of 11

Reply:
does this work in multiple languages?

Reply:
modified this to allow the months to be translatable